doc/api/path.md

```diff
@@ -416,6 +416,16 @@ added: v0.11.15
 The `path.posix` property provides access to POSIX specific implementations
 of the `path` methods.
 
+### path.posix.normalize(path)
+The `path.posix.normalize()` method will not attempt to convert / (Windows) to 
+\ (POSIX), as / is not recognized by POSIX as a valid directory separator.  
```

Comment:
I think the comment may have these backwards. The windows separator `\` is not recognized by POSIX as a valid separator.